Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -213,4 +213,19 @@ class CreditCardHelper: TabContentScript {
                        category: .webview)
         }
     }
+
+    // Note: document.activeElement.blur() is used to remove the focus from the
+    // currently focused element on a web page. When an element is focused,
+    // it typically has a visual indication such as a highlighted border or change in appearance.
+    // The reason we do it is because after pressing done the focus still remains in WKWebview
+    static func blurActiveElement(tabWebView: WKWebView,
+                                  logger: Logger) {
+        let fillCreditCardInfoCallback = "document.activeElement.blur()"
+        tabWebView.evaluateJavascriptInDefaultContentWorld(fillCreditCardInfoCallback) { _, err in
+            guard let err = err else { return }
+            logger.log("Unable to go previous field: \(err)",
```

Comment:
nit, I think it's preferreable with no abbreviation 🤷‍♀️ 
```suggestion
        tabWebView.evaluateJavascriptInDefaultContentWorld(fillCreditCardInfoCallback) { _, error in
            guard let error = error else { return }
            logger.log("Unable to go previous field: \(error)",
```

Reply:
Good call, performed a similar cleanup to other two methods as well